tests/test_api.py

```diff
@@ -99,7 +89,7 @@ async def test_malformed_document(joint_fixture: JointFixture):  # noqa: F811
 
 
 @pytest.mark.asyncio
-async def test_search(joint_fixture: JointFixture):  # noqa: F811
+async def test_search(joint_fixture: JointFixture, reset_state):
```

Comment:
```suggestion
async def test_search(joint_fixture: JointFixture):
```

You do not need to specify `reset_state` here and in all the other places when it is declared as autouse.

Reply:
Part of [f3a2bd5](https://github.com/ghga-de/mass/pull/9/commits/f3a2bd5831efe9215f5d521e4812b2dc84b03bef)